app/uk/gov/hmrc/customs/declaration/controllers/FileTransmissionNotificationController.scala

```diff
@@ -49,15 +48,15 @@ class FileTransmissionNotificationController @Inject() (successToXmlNotification
           case JsSuccess(callbackBody, _) => callbackBody match {
```

Comment:
now that you have a a single Xml transform function I think you can collapse bothe branches of ` case JsSuccess(callbackBody, _) => callbackBody match {` into one as the only difference id the XML transformation

Reply:
Done